aep/general/0151/aep.md.j2

```diff
@@ -7,41 +7,89 @@ return some kind of promise to the user, and allow the user to check back in
 later.
 
 The long-running request pattern is roughly analogous to a [Future][] in Python
-or Java, or a [Node.js Promise][]. Essentially, the user is given a token that
-can be used to track progress and retrieve the result.
+or Java, or a [Node.js Promise][]: the user is given a token that can be used to
+track progress and retrieve the result.
 
 ## Guidance
 
 Operations that might take a significant amount of time to complete **should**
-return a `202 Accepted` response along with an `Operation` resource that can be
-used to track the status of the request and ultimately retrieve the result.
+return an `Operation` resource that can be used to track the status of the
+request and ultimately retrieve the result.
 
-Any single operation defined in an API surface **must** either _always_ return
-`202 Accepted` along with an `Operation`, or _never_ do so. A service **must
-not** return a `200 OK` response with the result if it is "fast enough", and
-`202 Accepted` if it is not fast enough, because such behavior adds significant
-burdens for clients.
+A single method defined in an API surface **must** either _always_ return an
+`Operation`, or _never_ do so. As an example, a long-running operation **must
+not** return a response with the result of the operation if the operation
+completes quickly.
 
-**Note:** User expectations can vary on what is considered "a significant
-amount of time" depending on what work is being done. A good rule of thumb is
-10 seconds.
+### Interface Definitions
 
-### Operation representation
+{% tab proto %}
 
-The response to a long-running request **must** be an [`Operation`][Operation].
+When using protocol buffers, the common component
+[`aep.api.Operation`][aep.api.Operation] is used.
```

Comment:
**question:** Is this undefined, or am I failing to find it? cf [GitHub](https://github.com/aep-dev/aep/tree/main/proto/aep-api/aep/api) and [Buf](https://buf.build/aep/api/docs/main:aep.api)

Reply:
I think this will be fixed after https://github.com/aep-dev/aep-components/pull/8 - but @rofrankel could confirm.